images/manageiq-base-worker/container-assets/extract-vmware-vddk

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd /vddk
+filename=$(ls VMware-*)
+
+# Skip the install if the VDDK library is not found
+[[ -f $filename ]] && tar -zxvf VMware-* || mkdir vmware-vix-disklib-distrib
```

Comment:
Looked weird to create an empty directory if there were no vddk files but I see you need it so that https://github.com/ManageIQ/manageiq-pods/pull/666/files#diff-cd386fc86f89ae5eef0d1a52e06f621b5d8286f39ddc055e1cfaa7229b6316d9R20 will work

Reply:
Yeah, I don't really like it either, but it seemed like the cleanest approach